include/vctr_utils/vctr_juce_helpers.h

```diff
@@ -56,26 +56,26 @@ namespace vctr::juce_helpers
     @see VctrBase::getAVX
  */
 template <is::anyVctr Src>
-auto toSIMDRegister (const Src& src)
+auto toSIMDRegister (const Src& src, size_t i = 0)
```

Comment:
Could you please document the second parameter?

Also, although kind of obvious from the method name, I'd like formal documentation of the return value since we are using `auto` here.

Reply:
> Loads the first elements of src into a juce::SIMDRegister.

This needs to be reformulated

Reply:
Thx, I updated the comment